components/floatingactionbutton/index.css

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-FloatingActionButton {
+  --spectrum-floating-action-button-size: var(--spectrum-component-height-200);
+  --spectrum-floating-action-button-icon-size: var(--spectrum-workflow-icon-size-200);
+  --spectrum-floating-action-button-padding: var(--spectrum-component-pill-edge-to-visual-only-200);
+  --spectrum-floating-action-button-margin: var(--spectrum-spacing-200);
+  --spectrum-floating-action-button-drop-shadow-x: var(--spectrum-drop-shadow-x);
+  --spectrum-floating-action-button-drop-shadow-color: var(--spectrum-floating-action-button-shadow-color); 
```

Comment:
@mlogsdon18 --spectrum-floating-action-button-shadow-color this is undefined
Can you please check this. This is breaking the build